include/sundials/sundials_math.h

```diff
@@ -44,8 +44,24 @@ extern "C" {
  * SUNRabs calls the appropriate version of abs
  *
  * SUNRexp calls the appropriate version of exp
+ * 
+ * SUNRlog calls the appropriate version of log
+ * 
+ * SUNRsinh calls the appropriate version of sinh
+ * 
+ * SUNRcosh calls the appropriate version of cosh
  *
  * SUNRceil calls the appropriate version of ceil
+ * 
+ * SUNRfloor calls the appropriate version of floor
+ * 
+ * SUNRround calls the appropriate version of round
+ * 
+ * SUNIceil calls the appropriate version of ceil and returns sunindextype
+ * 
+ * SUNIfloor calls the appropriate version of floor and returns sunindextype
+ * 
+ * SUNIround calls the appropriate version of round and returns sunindextype
  * -----------------------------------------------------------------
```

Comment:
I don't have a preference on this.  

Generally, we code our functions to the configured precision, in which case `sunindextype` makes sense.  However, these functions were only added for this specific use case, and LSRKStep only allows the number of requested stages to be `int` (not the potentially longer `sunindextype`), so changing them to `int` would also make sense.  

I suppose the relevant question is whether anyone else can foresee a use for any of these in the future (in which case they could be adjusted appropriately).  An alternate option would be to move these inside `arkode_lsrkstep.c` instead of putting them in `sundials_math.h`, in which case they could be `int` and future developers could still create something different in `sundials_math.h`.

Reply:
I think making them internal is a good idea since they're not needed in other places.